Index: src/chap02/sec03/exam04.java
===================================================================
diff --git a/src/chap02/sec03/exam04.java b/src/chap02/sec03/exam04.java
new file mode 100644
--- /dev/null	
+++ b/src/chap02/sec03/exam04.java	
@@ -0,0 +1,11 @@
+package chap02.sec03;
+
+public class exam04 { //정수 타입의 연산 LongOperationExample.java
+    public static void main(String[] args) {
+        byte value1 = 10;
+        int value2 = 100;
+        long value3 = 1000L;
+        long result = value1 + value2 + value3; //가장 큰 허용범위를 가진 타입으로 자동 변환됨
+        System.out.println(result);
+    }
+}
Index: src/chap02/sec03/exam06.java
===================================================================
diff --git a/src/chap02/sec03/exam06.java b/src/chap02/sec03/exam06.java
new file mode 100644
--- /dev/null	
+++ b/src/chap02/sec03/exam06.java	
@@ -0,0 +1,7 @@
+package chap02.sec03;
+
+public class exam06 { //문자열 결함 연산 StringConcatExample.java
+    public static void main(String[] args) {
+
+    }
+}
Index: src/chap02/sec03/exam05.java
===================================================================
diff --git a/src/chap02/sec03/exam05.java b/src/chap02/sec03/exam05.java
new file mode 100644
--- /dev/null	
+++ b/src/chap02/sec03/exam05.java	
@@ -0,0 +1,32 @@
+package chap02.sec03;
+
+public class exam05 { //연산식에서 자동 타입 변환 OperationPromotionExample.java
+    public static void main(String[] args) {
+        byte byteValue1 = 10;
+        byte byteValue2 = 20;
+        //byte byteValue3 = byteVlaue1 + byteValue2; //컴파일 에러 byte < byte
+        int intValue1 = byteValue1 + byteValue2;
+        System.out.println(intValue1);
+
+        char charValue1 = 'A';
+        char charValue2 = 1;
+        //char charValue3 = charValue1 + charValue2; //컴파일 에러 문자열 - possible lossy conversion from int to char
+        int intValue2 = charValue1 + charValue2;
+        System.out.println("유니코드 = " + intValue2);
+        System.out.println("출력문자 = " + (char)intValue2);
+
+        int intValue3 = 10;
+        int intValue4 = intValue3/4;
+        System.out.println(intValue4);
+
+        int intValue5 = 10;
+        //int intValue6 = 10 / 4.0; //컴파일 에러
+        double doubleValue = intValue5 / 4.0;
+        System.out.println(doubleValue);
+
+        int x = 1;
+        int y = 2;
+        double result = (double) x / y; //(double) (x / y)일 경우 0.0 결과값 -> (x / y)먼저 계산되어 0 -> (double) 적용되어 0.0
+        System.out.println(result);
+    }
+}
